docs/tool_jcmd.md

```diff
@@ -63,5 +67,9 @@ The tool uses the Attach API, and has the following limitations:
 
 For more information about the Attach API, including how to enable and secure it, see [Java Attach API](attachapi.md).
 
+## See Also
+
+-  [What's new in version 0.44.0](version0.44.md#vmid-query-in-the-jcmd-tool-enhanced)
```

Comment:
Do we often refer back to the What's New from specific topics? Not sure this is necessary if all the information is in the topic.

Reply:
This is related to an issue that one of our clients has raised in https://github.com/eclipse-openj9/openj9-docs/issues/1224. I am working on this separately. It helps the user to identify the release when a change was introduced and whether it is applicable to user who might be on a different version. Maybe, I can move this change to that issue?

Reply:
Keeping it here is fine now I know the reason for it.